### PR TITLE
Started the fletching section

### PR DESCRIPTION
**Pull request by ConradicalMel**
Not close to done. Need to leave for Christmas dinner so I thought I'd commit and PR first to get us synced up.

### CODE REVIEW

File: src/main/java/thestonedturtle/bankedexperience/data/Activity.java [hunk 1]
```diff
@@ -598,6 +598,58 @@
...
+	 */
+	// Logs
+	F_LOGS_HEADLESS_ARROWS(ItemID.ARROW_SHAFT, "Arrow shaft", 1, 5,
+		ExperienceItem.F_LOGS, null, null),
+	F_LOGS_SHORTBOW_U(ItemID.SHORTBOW_U, "Shortbow (u)", 5, 5,
+		ExperienceItem.F_LOGS, null, null),
+	F_LOGS_LONGBOW_U(ItemID.LONGBOW_U, "Longbow (u)", 10, 10,
+		ExperienceItem.F_LOGS, null, null),
+	// Oak Logs
+	F_OAK_LOGS_HEADLESS_ARROWS(ItemID.ARROW_SHAFT, "Arrow shaft", 15, 10,
```
Comment (TheStonedTurtle) [inline src/main/java/thestonedturtle/bankedexperience/data/Activity.java]: Pretty sure you can remove `_LOGS_` from here

**Comment by TheStonedTurtle** [comment]
Sorry for the open/close spam, this project should have travis CI setup so if your test doesn't pass please fix it (CheckStyle probably)

**Comment by ConradicalMel** [comment]
Ok it should be good to go now, don't worry about the spam lol

**Comment by TheStonedTurtle** [comment]
Please do a `git rebase -i master` and remove the duplicate commit (8022706)

**Pull request merged**